developer-docs-site/docs/concepts/delegated_staking.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Delegated Staking"
+slug: " Delegated staking"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Delegated Staking
+
+## Delegated Staking on the Aptos blockchain
+
+:::tip Consensus We strongly recommend that you read the consensus section of Aptos Blockchain Deep Dive before proceeding further. :::
+
+Delegated staking is an extension of the staking protocol on Aptos. A delegation pool abstracts the stake owner to an entity capable of collecting stake from delegators and adding it on their behalf to the native stake pool attached to the validator. This means that multiple entities can stake to a stake pool to reach the minimum requirement for the validator to join the validator set. Delegators can add stake to an inactive pool, but the delegation pool will not earn rewards until it is active. 
+
+:::danger Delegation pools are permissionless and anyone can stake to the validator. You cannot change the type of stake pool once it's created. For full ownership of the stake pool, see [Staking](../concepts/staking.md)
+::: 
+
+See [delegation_pool.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/delegation_pool.move) 
+
+A delegation pool can be initialized without adding stake. A resource account is created along with an empty stake pool it owns from that point on.This account will manage the stake of the underlying stake pool on behalf of the delegators by forwarding their stake-management operations to it (add, unlock, reactivate, withdraw) while the resource account cannot be directly accessed nor externally owned.
```

Comment:
"A delegation pool can be initialized without adding stake." -> "A delegation pool can be initialized with zero stake."